Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/ui/details/CovidCertificateDetailsFragment.kt

```diff
@@ -63,13 +64,8 @@ class CovidCertificateDetailsFragment : Fragment(R.layout.fragment_covid_certifi
     private fun FragmentCovidCertificateDetailsBinding.onCertificateReady(
         testCertificate: TestCertificate
     ) {
-        qrCodeCard.subtitle.text = getString(
-            R.string.detail_green_certificate_card_subtitle,
-            testCertificate.sampleCollectedAt.toShortDayFormat(),
-            testCertificate.sampleCollectedAt.toShortTimeFormat(),
-        )
         name.text = testCertificate.run { "$lastName, $firstName" }
-        birthDate.text = testCertificate.dateOfBirth.toDayFormat()
+        birthDate.text = testCertificate.dateOfBirth.toString(dayOfBirthFormatter)
```

Comment:
is it a birthdate specific format ? can we create an extension function for that if it has to be for all birthdates?
`toBirthdateFormat()`